editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-spy-wrapper-third-party.spec.browser.tsx

```diff
@@ -9,10 +9,10 @@ jest.mock('../../../core/es-modules/package-manager/built-in-dependencies-list',
     {
       moduleName: '@react-three/fiber',
       nodeModule: {
-        ...ReactThreeFiber,
-        default: ReactThreeFiber,
+        ...MockReactThreeFiber,
```

Comment:
babel only lets you use things from out of scope which are named mock* (case insensitive)